tests/test_demo_extractors.py

```diff
@@ -0,0 +1,60 @@
+import os
+import unittest
+
+from maco import cli
+from maco.collector import Collector
+
+
+class TestDemoExtractors(unittest.TestCase):
+    def test_complex(self):
+        path_file = os.path.normpath(
+            os.path.join(__file__, "../data/trigger_complex.txt")
+        )
+        collector = Collector(os.path.join(__file__, "../../demo_extractors"))
+        self.assertEqual(
+            set(collector.extractors.keys()),
+            {
+                "Elfy",
+                "Nothing",
+                "Complex",
+                "LimitOther",
+            },
+        )
+
+        with open(path_file, "rb") as stream:
+            ret = cli.process_file(
+                collector,
+                path_file,
+                stream,
+                pretty=True,
+                force=False,
+                include_base64=False,
+            )
+        self.assertEqual(
+            ret,
+            {
+                "Complex": {
+                    "family": "complex",
+                    "version": "5",
+                    "decoded_strings": ["Paradise", "Complex"],
```

Comment:
Change to: "decoded_strings": ["Complex", "Paradise"]
based on sorted output from previous comment.